packages/printable-itinerary/src/PrintableItinerary.story.tsx

```diff
@@ -40,6 +41,13 @@ export const WalkOnlyItinerary = () => (
     LegIcon={TriMetLegIcon}
   />
 );
+export const OTP24Itinerary = () => (
```

Comment:
Can we add a comment or re-title this one, it's hard to tell what it's testing for!